assembl/static2/js/app/reducers/adminReducer/adminSections.js

```diff
@@ -67,32 +67,16 @@ export const sectionsById = (state: Map<string, Map> = Map(), action: ReduxActio
       return aOrder - bOrder;
     });
     const idToDelete = state.getIn([action.id, 'id']);
-    let newState = Map();
+    let newState = sections;
     let count = -1;
     sections.forEach((s) => {
       if (s.get('id') !== idToDelete && s.get('type') !== 'ADMINISTRATION' && !s.get('toDelete')) {
         count += 1;
-        const sectionInfo = Map({
-          toDelete: false,
-          isNew: s.get('isNew'),
-          order: count,
-          id: s.get('id'),
-          titleEntries: s.get('titleEntries'),
-          url: s.get('url'),
-          type: s.get('type')
-        });
-        newState = newState.set(s.get('id'), sectionInfo);
+        newState = newState.setIn([s.get('id'), 'order'], count);
```

Comment:
much better :) forget my previous comment https://github.com/assembl/assembl/pull/250/commits/9a7e4fd60bffeec6ecd2ce6cc13929dd6937d06e#r154046378

Reply:
if you do a review commit by commit, please compare with the latest diff of the PR before posting the comment ;)

Reply:
good idea, I had tried without success on previous branches but now that you do a bit smaller PRs they load fully in github :p